runbotics/runbotics-scheduler/src/websocket/bot/process-launch/bot-process-instance.service.ts

```diff
@@ -37,14 +37,14 @@ export class BotProcessService {
         await queryRunner.connect();
         await queryRunner.startTransaction();
         try {
-            const incrementedProcessInstance = await this.incrementExecutionCount(processInstance, newProcessInstance, bot);
-            const dbProcessInstance = await queryRunner.manager.findOne(ProcessInstanceEntity, { where: { id: incrementedProcessInstance.id } });
+            const updatedLastRunTimeInProcessInstance = await this.updateLastRunTime(processInstance, newProcessInstance, bot);
+            const dbProcessInstance = await queryRunner.manager.findOne(ProcessInstanceEntity, { where: { id: updatedLastRunTimeInProcessInstance.id } });
 
             await queryRunner.manager.createQueryBuilder()
                 .insert()
                 .into(ProcessInstanceEntity)
-                .values({ ...incrementedProcessInstance })
-                .orUpdate(getProcessInstanceUpdateFieldsByStatus(incrementedProcessInstance.status, dbProcessInstance?.status), ['id'])
+                .values({ ...updatedLastRunTimeInProcessInstance })
+                .orUpdate(getProcessInstanceUpdateFieldsByStatus(updatedLastRunTimeInProcessInstance.status, dbProcessInstance?.status), ['id'])
```

Comment:
take a look at `getProcessInstanceUpdateFieldsByStatus`, imo you should adjust object keys which will be updated